<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>315</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>315</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>315</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LettersVoice</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../f68dbafa/document-speech.png</normaloff>../f68dbafa/document-speech.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>109543</width>
     <height>109543</height>
    </size>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>640</width>
      <height>305</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>640</width>
      <height>305</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>640</width>
      <height>305</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <widget class="QWidget" name="document_tab">
     <attribute name="icon">
      <iconset>
       <normaloff>../f68dbafa/document-speech.png</normaloff>../f68dbafa/document-speech.png</iconset>
     </attribute>
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QPushButton" name="btnChooseFile">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>10</y>
        <width>625</width>
        <height>36</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Choose file ...</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>611</width>
        <height>47</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>20</horstretch>
           <verstretch>20</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>Choose the document language</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <property name="indent">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbSpeechLanguage_1">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>32</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select language for live speeching&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="currentText">
          <string>en</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="maxCount">
          <number>7151</number>
         </property>
         <property name="insertPolicy">
          <enum>QComboBox::InsertAtTop</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>8</width>
           <height>8</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>en</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ru</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btnChooseFolder">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>120</y>
        <width>625</width>
        <height>36</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="text">
       <string>Choose folder to save speech file ...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPerform">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>225</y>
        <width>625</width>
        <height>36</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(48,140,198);</string>
      </property>
      <property name="text">
       <string>Perform file speeching</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="live_tab">
     <attribute name="icon">
      <iconset>
       <normaloff>../cc356acc/live-speech.png</normaloff>../cc356acc/live-speech.png</iconset>
     </attribute>
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QTextEdit" name="tbLive">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>5</y>
        <width>625</width>
        <height>200</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Live speeching while typing&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="toolTipDuration">
       <number>5</number>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbSpeechLanguage_2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>230</y>
        <width>75</width>
        <height>32</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>32</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select language for live speeching&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="currentText">
       <string>en</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="maxCount">
       <number>7151</number>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::InsertAtTop</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>8</width>
        <height>8</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>en</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ru</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="btnMuteUnmute">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>230</y>
        <width>32</width>
        <height>32</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Mute/Unmute&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="toolTipDuration">
       <number>5</number>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../cecca979/unmuted.png</normaloff>../cecca979/unmuted.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionRussian"/>
   </widget>
   <addaction name="menuLanguage"/>
  </widget>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionRussian">
   <property name="text">
    <string>Russian</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
